docs/verify/simulators.mdx

```diff
@@ -1,11 +1,11 @@
 ---
-title: Simulators overview
-description: Overview of simulators
+title: IBM Quantum simulators
+description: Overview of IBM Quantum cloud-based simulators
 ---
-# Simulators overview
+# IBM Quantum simulators
 
 <Admonition type="note">
-This page gives details about the IBM Quantum cloud-based simulators. For information about the Qiskit built-in simulator, see the [Python-based simulators page in the API reference.](https://docs.quantum-computing.ibm.com/api/qiskit/providers_basicaer)
+This page gives details about the IBM Quantum cloud-based simulators. For information about the Qiskit built-in simulator, see the [Python-based simulators page in the API reference.](https://docs.quantum-computing.ibm.com/api/qiskit/providers_basicaer) You can also use the [Qiskit reference primitives](simulate-with-primitives) for local statevector simulation.
```

Comment:
Is this still relevant?

> For information about the Qiskit built-in simulator, see the [Python-based simulators page in the API reference.](https://docs.quantum-computing.ibm.com/api/qiskit/providers_basicaer)

If so, the URL should be a relative link to `../api/qiskit/providers_basicaer`

Reply:
good catch!